### ISSUE TITLE
Uncaught TypeError: Cannot read property 'isVisible' of undefined

### ISSUE THREAD

**Issue by k4zuk0** [defunkt/Zen#108]
Hi,

I'm having a glitchy behavior, that I think is caused by Nuclide.
The only thing that happens when I toggle Zen is that the Tabs bar disappears, nothing else changes (the editor stays the same, tree view still visible, right tool bar too, etc). Basically nothing happens to any pane other than the tabs one.

I just updated it to 1.17, but the same thing happened with the previous version (1.16.4 I think).

[Enter steps to reproduce:]

1. Open Atom
2. Toggle Zen

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12
**Thrown From**: [Zen](https://github.com/defunkt/zen) package 0.17.0

### Stack Trace

Uncaught TypeError: Cannot read property 'isVisible' of undefined

```
At /Users/kazu/.atom/packages/Zen/lib/zen.coffee:150

TypeError: Cannot read property 'isVisible' of undefined
    at Object.toggle (/packages/zen/lib/zen.coffee:150:17)
    at /packages/zen/lib/zen.coffee:53:59)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:1)
    at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:1)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:1)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:1)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:1)
```

### Commands

```
     -0:26.3.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling)
  2x -0:17.7.0 core:close (div.markdown-preview.native-key-bindings)
     -0:10.6.0 editor:checkout-head-revision (input.hidden-input)
     -0:08.5.0 command-palette:toggle (input.hidden-input)
     -0:05.8.0 editor:consolidate-selections (input.hidden-input)
     -0:05.8.0 core:cancel (input.hidden-input)
  2x -0:05.2.0 zen:toggle (input.hidden-input)
```

### Non-Core Packages

```
activate-power-mode 1.4.0 
angularjs 0.4.0 
atom-autocomplete-php 0.22.2 
atom-beautify 0.29.24 
atom-clock 0.1.7 
atom-ternjs 0.18.3 
atomify 0.3.0 
autocomplete-modules 1.6.10 
build 0.67.0 
build-make 0.13.0 
busy 0.7.0 
color-picker 2.2.5 
emmet 2.4.3 
file-icons 2.1.6 
git-control 0.9.0 
goto 1.8.3 
hey-pane 1.0.0 
hyperclick 0.0.0 
intentions 1.1.2 
language-hack 0.1.1 
linter 2.1.4 
merge-conflicts 1.4.5 
minimap 4.28.2 
minimap-cursorline 0.2.0 
minimap-pigments 0.2.2 
nuclide 0.228.0 
octocat-syntax 1.3.1 
php-integrator-annotations 1.1.1 
php-integrator-autocomplete-plus 1.5.0 
php-integrator-base 2.1.12 
php-integrator-call-tips 1.2.1 
php-integrator-linter 1.3.0 
php-integrator-navigation 1.1.2 
php-integrator-refactoring 1.3.1 
php-integrator-tooltips 1.1.5 
php-twig 4.0.0 
pigments 0.39.1 
sort-lines 0.14.0 
term3 0.22.1 
todo-show 1.11.0 
tool-bar 1.1.0 
travis-ci-status 1.3.1 
vim-mode 0.66.0 
Zen 0.17.0 
```

**Comment by braver** [comment]
I think the new docks api broke this. I just published `v0.18.0`, hopefully it fixes this, if not let me know.

**Comment by anthony-moyart** [comment]
It is indeed fixed, thanks and well done!
